src/java/org/apache/cassandra/service/accord/AccordSafeCommand.java

```diff
@@ -120,9 +121,9 @@ public Command original()
         return original;
     }
 
-    public SavedCommand.Writer diff()
+    public Journal.CommandUpdate change()
```

Comment:
is it an update, a change, some changes, a diff...? Let's pick a name and move to it everywhere.

Reply:
I have renamed to `update` for now, but I am not a big fan of it since it is also a verb. I have a follow-up patch that cleans it all up, hope this is OK for interim.